src/dialog/snippets/index.html

```diff
@@ -0,0 +1,42 @@
+<div class="mdl-dialog mdl-transition-in" aria-label="Mango (Fruit)">
+  <form>
+  <div class="mdl-toolbar">
+    <div class="mdl-toolbar-tools">
+      <h2>Mango (Fruit)</h2>
+      <span flex></span>
+      <div class="mdl-button" class="mdl-icon-button" ng-click="cancel()">
+        <div class="mdl-icon" mdl-svg-src="img/icons/ic_close_24px.svg" aria-label="Close dialog"></div>
```

Comment:
We should use font icons instead of images. It is bundled in the material ui lite.